app/src/main/java/io/homeassistant/companion/android/vehicle/MainVehicleScreen.kt

```diff
@@ -137,78 +154,30 @@ class MainVehicleScreen(
     }
 
     override fun onGetTemplate(): Template {
-        val listBuilder = ItemList.Builder()
+        var listBuilder = ItemList.Builder()
+        var favoritesGrid: ItemList.Builder? = null
         if (favoritesList.isNotEmpty()) {
```

Comment:
so this poses a question do we switch to grid view instead of list view for the home screen without favorites then? do we think that would give access to more domains on the screen as opposed to the list view?

Reply:
> do we switch to grid view instead of list view for the home screen without favorites then?

I like that, less changes when you add a favorite.

> do we think that would give access to more domains on the screen as opposed to the list view?

Depends on the vehicle but at least for devices that are wider it seems like it? For example desktop head unit 6 grid items vs 4 list items without scrolling.

Reply:
yea a lot of cars I saw were wider landscape than portrait too

Reply:
thinking more about this I think we still need to check if the server has favorites or not

Reply:
Done in the latest commit and I think it looks a lot cleaner. At least there are more items. In my car `ListTemplate` shows 10 items but `GridTemplate` can show 15 items in the same amount of space.